src/utils.ts

```diff
@@ -36,8 +36,9 @@ export const messages = {
   getProviderInvalidRequest: `GetProvider request is not valid`,
 };
 
-export const fetchFile = (token: string, fileName: string) => fetch(`${token}/${fileName}.json`)
-  .then((response) => response.json());
+export const fetchFile = (token: string, fileName: string, disableCache?: boolean) => {
+  const timeStamp = disableCache ? `?=${new Date().getTime()}`: '';
```

Comment:
Date.now suppose to faster and nicer to look on

Reply:
done